src/config/dynamic-theme-fixes.config

```
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
+    background-color: rgb(93, 122, 136);
```

```suggestion
    background-color: ${#6f92a3} !important;
```
What about changing it a little bit.

---

I agree \o/